muddy/muddy_macros/src/lib.rs

```diff
@@ -129,10 +129,10 @@ impl InPlaceDecrypter {
             });
             #[cfg(windows)]
             // language=cmd
-            eprintln!(r"set "{env}={key}"");
+            eprintln!(r#"set "{env}={key}""#);
             #[cfg(not(windows))]
             // language=sh
-            eprintln!(r"{env}='{key}'");
+            eprintln!(r#"{env}={key}"#);
```

Comment:
Hey! Sorry I was a bit unclear. What I meant was that this change should be reverted since it's unrelated to the problem (and clippy complains). If you revert this change I'll happily merge the changes to windows `eprintln!()` fix just above
 
```suggestion
            eprintln!(r"{env}='{key}'");
```

Reply:
just merge my pull request